cookbook-react/recipes/inputs/barcelona/styles.module.scss

```diff
@@ -0,0 +1,106 @@
+$input-border-default-color: #B5B5B6;
+$input-border-focus-color: #0043CD;
+$input-border-error-color: #EA4453;
+$input-border-valid-color: #34DC85;
+
+$disabled-text-color: #525256;
+$disabled-background-color: #E1E1E1;
+$disabled-border-color: #E8E8E8;
+
+$placeholder-color: #9C9C9C;
+
+$white: #FFF;
+$text-color: #08080D;
+$label-color: #525256;
+
+
+.input-wrapper {
+  margin-bottom: 5px;
```

Comment:
I don't think the input should have external margins. If it isn't external ignore this comment.